src/test/ui/generic-associated-types/issue-81069.rs

```diff
@@ -0,0 +1,24 @@
+// check-pass
```

Comment:
Wait, this test is named #81069. And doesn't even use GATs, so why is it in this directory?

Reply:
Ah this was supposed to be `91069`, looks like I fatfingered it